conversion/src/main.rs

```diff
@@ -0,0 +1,85 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+//! This code serves the purpose of converting to a dusk_plonk suitable format
+//! the Trusted Setup response file computed using this code:
+//!
+//! https://github.com/dusk-network/powersoftau.git
+
+use dusk_bls12_381::{G1Affine, G2Affine};
+use std::{fmt::Error, fs, fs::File, io::Write};
+
+// The maximum number of tau powers computed in the Dusk Trusted Setup
```

Comment:
// The maximum number of tau powers **in G1** computed in the Dusk Trusted Setup